### ISSUE TITLE
Accessibility: MAS 42A:  In the high contrast white theme, there is no border and download arrow  is missing for download link

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#10]
#A11yMAS
#MAS42A 
#Wipro 
#vscode-website
#Firefox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Prerequisite:**
**_1. Change system setting to High contrast White_**
    Go to Settings-->Ease of Access-->High Contrast.
    Select "High Contrast White" from the "Choose a theme" dropdown. Click on Apply.

**Repro Steps:**
1. Open https://code.visualstudio.com/ in Edge.
2. Navigate to the "Download" link at top RHS in the  header of the page .
3. Observe the "Download" link border and download arrow which is missing in High contrast white theme.

Note:Same issue is observed for the below link
https://code.visualstudio.com/docs
http://code.visualstudio.com/Download
http://code.visualstudio.com/insiders
https://code.visualstudio.com/blogs
https://code.visualstudio.com/updates
https://code.visualstudio.com/docs/supporting/faq
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/introvideos/overview
https://code.visualstudio.com/docs/editor/accessibility
https://code.visualstudio.com/docs/customization/userandworkspace
https://code.visualstudio.com/docs/languages/overview
https://code.visualstudio.com/docs/setup/setup-overview
https://code.visualstudio.com/docs/runtimes/nodejs

**Actual Result:**
Issue1:The "Download " link is not having the proper border in high contrast white  theme.

Issue2:
Download arrow is missing in high contrast white  theme.

**Expected Result:**
Issue1:The "Download " link  should have proper border in high contrast white  theme as visible in normal mode.

Issue2:Download arrow  should be visible in high contrast white  theme as visible in normal mode.

**User Impact:**
The persons with color blindness and low vision will not be able to understand that it is a download button and may not use the application efficiently.
[No proper border for download link.zip](https://github.com/wiproaccess/vscode-website/files/686940/No.proper.border.for.download.link.zip)

**Comment by waderyan** [comment]
![image](https://cloud.githubusercontent.com/assets/2449568/26735673/b7e30486-4777-11e7-9e7d-f6574823b664.png)

**Comment by waderyan** [comment]
Fixed with f3feb4114be9ee9ae06f6b1da431834c316fb059

**Comment by wiproaccess** [comment]
The issue is verified hence adding the label 'closed' and closing the bug.
